src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -612,6 +613,15 @@
       (.addColLabel pivot-table idx))
     (doseq [idx pivot-measures]
       (.addColumnLabel pivot-table DataConsolidateFunction/SUM #_(get aggregation-functions idx DataConsolidateFunction/SUM) idx))
+    (doseq [[idx sort-setting] column-sort-order]
+      (let [setting (sort-setting {:ascending  STFieldSortType/ASCENDING
+                                   :descending STFieldSortType/DESCENDING})]
```

Comment:
nit — since it's not immediately obvious that `sort-setting` is a keyword, I think a `case` statement here might be a little more readable? just a suggestion, since I was slightly thrown by this initially
```suggestion
      (let [setting (case sort-setting
                      :ascending STFieldSortType/ASCENDING
                      :descending STFieldSortType/DESCENDING)]
```